pvlib/clearsky.py

```diff
@@ -195,7 +195,9 @@ def ineichen(time, location, linke_turbidity=None,
     
     cos_zenith = tools.cosd(ApparentZenith)
     
-    clearsky_GHI = cg1 * I0 * cos_zenith * np.exp(-cg2*AMabsolute*(fh1 + fh2*(TL - 1))) * np.exp(0.01*AMabsolute**1.8)
+    clearsky_GHI = ( cg1 * I0 * cos_zenith *
```

Comment:
would also change `clearsky_GHI` to all lower case (`clearsky_ghi`) for consistency.
Not sure, did you intended to change only names for the kw arguments (all api related) or all internal variable names too?